api/v1beta1/backuppolicy_types.go

```diff
@@ -83,7 +83,6 @@ type BackupPolicySpec struct {
 }
 
 //+kubebuilder:object:root=true
-//+kubebuilder:storageversion
 
```

Comment:
also set //+kubebuilder:deprecatedversion

Reply:
How about adding it in the next PR?

Reply:
I understand.